types/src/transaction/mod.rs

```diff
@@ -450,9 +460,10 @@ impl fmt::Debug for SignedTransaction {
             "SignedTransaction {{ \n \
              {{ raw_txn: {:#?}, \n \
              authenticator: {:#?}, \n \
+             bytes: {:#?}, \n \
              }} \n \
              }}",
-            self.raw_txn, self.authenticator
+            self.raw_txn, self.authenticator, self.bytes
```

Comment:
I wouldn't add this, it'll add a lot of noise

Reply:
yeah, don't print out raw bytes, we can print out the len

Reply:
Sounds good! Updated the code 😄